var/spack/repos/builtin/packages/sperr/package.py

```diff
@@ -37,11 +40,10 @@ def cmake_args(self):
 
         args = [
             self.define_from_variant("BUILD_SHARED_LIBS", "shared"),
-            self.define_from_variant("USE_ZSTD", "zstd"),
```

Comment:
Don't you still want this for when `+zstd`?

Reply:
Yes, this is needed when `+zstd`

Reply:
Thank you so much for pointing it out! Yes, it is still needed! I've brought it back! 

(Also, the CMake flag `USE_BUNDLED_ZSTD` is actually not needed since version 0.7.x, but leaving it there has no harm. I think it's OK to just leave it there?)

Reply:
Depends on how the software builds work but I don't see why not unless keeping it there causes a build failure.  You could also add a variant to enable/disable that feature for versions that use it with the `when` argument, as you've done with `zstd`.

Reply:
This is a great idea! I've added a variant to toggle it.